### PR TITLE
[WIP] Rails 5.0 

### PR DESCRIPTION
**Pull request by marten**
Describe your change here.

# Review checklist

- [ ] First, the most important one: is this PR small enough that you can actually review it? Feel free to just reject a branch if the changes are hard to review due to the length of the diff.
- [ ] If there are any migrations, will they the previous version of the app work correctly after they've been run (e.g. the don't remove columns still known about by ActiveRecord).
- [ ] If anything changed with regards to the public API, are those changes also documented in the `apiary.apib` file?
- [ ] Are all the changes covered by tests? Think about any possible edge cases that might be left untested.

### CODE REVIEW

File: Gemfile.next [hunk 1]
```diff
@@ -0,0 +1 @@
+Gemfile
```
Comment (marten) [inline Gemfile.next]: This is apparently what symlinks look like in Github's diffs. So note, this file is a symlink to the normal gemfile, not a file with this line in it.

File: .travis.yml [hunk 2]
```diff
@@ -8,9 +8,7 @@ before_install:
   - find spec -maxdepth 1 -type d | grep spec/ > tmp/curr_spec_dirs.txt
   - if grep -Fxvc -f spec/known_dirs.txt tmp/curr_spec_dirs.txt; then echo 'Detected unkown Spec directories, check the spec/known_dirs.txt!'; exit 1; fi
   - 'echo ''gem: --no-ri --no-rdoc'' > ~/.gemrc'
-
-install:
-  - bundle install --without development production --path=${BUNDLE_PATH:-vendor/bundle}
+  - if [ "$BUNDLE_GEMFILE" = "$PWD/Gemfile.next" ]; then rm -f Gemfile.lock; fi
```
Comment (marten) [inline .travis.yml]: If BUNDLE_GEMFILE is Gemfile.next, Bundler should also be making and using Gemfile.next.lock, so i'm confused about this change.

Comment (camallen) [reply .travis.yml]: This was super confusing to me as well, most update guides suggest running without a *.lock file but that is just silly. Anyway, turns out that the `Gemfile.next.lock` wasn't checked in and I sank way to long figuring that out :(

**Comment by camallen** [comment]
closing in favour of #3374

**Pull request closed without merging**